app/app/opportunities/create/page.tsx

```diff
@@ -1,11 +1,264 @@
-"use client"
+"use client";
+
+import Input from "@components/Input";
+import { Button } from "@components/ui/button";
+import { Cross1Icon, QuestionMarkCircledIcon } from "@radix-ui/react-icons";
+import Select from "@components/Select";
+import { Field, Form, Formik } from "formik";
+import * as Yup from "yup";
+import MemberBar, { AddMemberBar } from "../components/MemberBar";
+import Dialog from "@components/Dialog";
+import { useState } from "react";
+import ErrorMessage from "@components/ErrorMessage";
+
+const mockAdmins = [
+  {
+    id: 1,
+    photoUrl: "https://placekitten.com/200/200",
+    name: "Simon Huang",
+    tags: [
+      { text: "Owner", color: "yellow" },
+      { text: "Development", color: "blue" },
+    ],
+  },
+  {
+    id: 2,
+    photoUrl: "https://placekitten.com/201/201",
+    name: "Max von Storch",
+    tags: [{ text: "RnD", color: "green" }],
+  },
+];
+
+const mockScreeners = [
+  {
+    id: 3,
+    photoUrl: "https://placekitten.com/202/202",
+    name: "Emma Johnson",
+    tags: [{ text: "Legal and finance", color: "orange" }],
+  },
+  {
+    id: 4,
+    photoUrl: "https://placekitten.com/204/204",
+    name: "Oliver Davis",
+    tags: [{ text: "Community", color: "red" }],
+  },
+  {
+    photoUrl: "https://placekitten.com/206/206",
+    name: "Sophia Rodriguez",
+    tags: [{ text: "Development", color: "purple" }],
+  },
+];
+
+const mockMembers = [
+  {
+    key: "Simon Huang",
+    value: "1",
+  },
+  {
+    key: "Tim Baum",
+    value: "2",
+  },
+];
+
+enum Roles {
+  ADMIN = "Admin",
+  SCREENER = "Screener",
+}
+
 export default function CreateOpportunity() {
-    return (
-        <div className="flex flex-col gap-8 container">
-            <div className="flex flex-col gap-3">
-                <h1 className="font-thin text-6xl">Opportunity</h1>
-                <p>Configure a new opportunity</p>
-            </div>
-        </div>
-    );
+  return (
+    <div className="container mx-0 mb-12 flex flex-col gap-8 md:mx-24">
```

Comment:
I can't really see anything on the screenshot you provided, but since there was definitely something wrong with the layout I removed the `containter` class from the page and it now behaves better for me than before. Also I added `self-center`. Lmk what you think